src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import { FilteredTodos } from '../../enums/FilteredTodo';
+
+interface TodoFilterProps {
+  filterSelected: FilteredTodos;
+  setFilterSelected: (filterSelected: FilteredTodos) => void;
+}
+
+export const TodoFilter: React.FC<TodoFilterProps> = ({
+  filterSelected,
+  setFilterSelected,
+}) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        href="#/"
+        className={classNames('filter__link', {
+          selected: filterSelected === FilteredTodos.all,
+        })}
+        data-cy="FilterLinkAll"
+        onClick={() => setFilterSelected(FilteredTodos.all)}
+      >
+        {FilteredTodos.all}
+      </a>
+
+      <a
+        href="#/active"
+        className={classNames('filter__link', {
+          selected: filterSelected === FilteredTodos.active,
+        })}
+        data-cy="FilterLinkActive"
+        onClick={() => setFilterSelected(FilteredTodos.active)}
+      >
+        {FilteredTodos.active}
+      </a>
```

Comment:
You can create an array from your enum with `Object.values` and render links with map